torch/prototypes/nestedtensor/codegen/tensorextension.py

```diff
@@ -0,0 +1,190 @@
+import torch
+
+# These functions only take two inputs. An input and a output.
+# No scalars etc.
+def get_unary_functions():
+    return [
+        'abs',
+        'acos',
+        'asin',
+        'atan',
+        # 'atan2',
+        # 'byte',
+        'ceil',
+        # 'char',
+        'clamp',
+        # 'clone',
+        # 'contiguous',
+        'cos',
+        'cosh',
+        # 'cpu',
+        # 'cuda',
+        'digamma',
+        # 'div',
+        # 'double',
+        # 'dtype',
+        'erf',
+        'erfc',
+        'erfinv',
+        'exp',
+        'expm1',
+        # 'exponential_',
+        # 'float',
+        'floor',
+        # 'fmod',
+        'frac',
+        # 'half',
+        # 'hardshrink', #TODO: Not part of aten
+        # 'int',
+        'lgamma',
+        'log',
+        'log10',
+        'log1p',
+        'log2',
+        # 'long',
+        # 'lt',
+        # 'mvlgamma',
+        'neg',
+        'nonzero',
+        # 'polygamma',
+        # 'pow',
+        # 'prelu', # TODO: no prelu_out
+        'reciprocal',
+        # 'relu', # TODO: no relu_out
+        # 'remainder',
+        # 'renorm',
+        'round',
+        'rsqrt',
+        'sigmoid',
+        'sign',
+        'sin',
+        'sinh',
+        'sqrt',
+        # 'sub',
+        'tan',
+        'tanh',
+        'tril',
+        'triu',
+        'trunc']
+
+
+def get_binary_functions():
+    return [
+        'add',
+        'mul',
+        'sub',
+        'div',
+    ]
+
+
+def get_comparison_functions():
+    return [
+        'eq',
+        'ge',
+        'gt',
+        'le',
+        'ne',
+    ]
+
+
+def set_function(module, cls, tfunc, func):
+    def _gen_func(tfunc):
+        orig_tfunc = getattr(torch, tfunc)
+
+        def _func(*args, **kwargs):
```

Comment:
You want an @wraps(original_tfunc) so that the documentation and function name is preserved.